### PR TITLE
Comitting the supplied resources for Iotix

### LINKED ISSUE #7161: Iotix: Minor improvements and addition of style variations
I have made some improvements and added 3 style variations to **Iotix** and I am listing here what was changed as recommended by @mikachan:

- Replaced the theme screenshot since the previous one was a bit blurry. It also reflects the changes made to the home template.
- I added the hero image `Hero-Image.png` and the 404 template one `404_Image.png`. They weren’t showing up when I installed the theme. And it seems like this issue has come up before with the theme.
- Made spacing adjustments to the Home template and global spacing (header, footer, side margins, etc) so it would scale better to mobile.
- Made adjustments to the colors, added an extra color `#272542` for the Cover overlays, so that the style variations would work nicely.
- Added 3 style variations: two dark, one light.
- Added new fonts: Figtree, IBM Plex Serif, Playfair Display and Manrope.

Here's the zip file: [iotix.zip](https://github.com/Automattic/themes/files/11870962/iotix.zip)

### PR DESCRIPTION
**Pull request by pbking**
Addresses #7161

### CODE REVIEW

File: iotix/parts/footer.html [hunk 1]
```diff
@@ -1 +1,51 @@
-<!-- wp:pattern {"slug":"iotix/footer"} /-->
\ No newline at end of file
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"5vh","right":"5vw","bottom":"10vh","left":"5vw"}}},"layout":{"type":"constrained"}} -->
```
Comment (pbking) [inline iotix/parts/footer.html]: Looks like this footer content moved from a pattern to this template part.

I think the content needs to go back to the pattern.

File: iotix/patterns/404.php [hunk 2]
```diff
@@ -1,32 +1,37 @@
...
-<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- wp:spacer {"height":"40px"} -->
+<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
 <!-- /wp:spacer -->
 
-<!-- wp:heading {"textAlign":"left","level":1,"fontSize":"x-large"} -->
-<h1 class="has-text-align-left has-x-large-font-size" id="oops-that-page-can-t-be-found"><?php echo esc_html__( 'Oops! That page can’t be found.', 'iotix' ); ?></h1>
+<!-- wp:group {"layout":{"type":"default"}} -->
+<div class="wp-block-group"><!-- wp:heading {"textAlign":"left","fontSize":"x-large","anchor":"oops-that-page-can-t-be-found"} -->
+<h2 class="wp-block-heading has-text-align-left has-x-large-font-size" id="oops-that-page-can-t-be-found">Page not found.</h2>
```
Comment (pbking) [inline iotix/patterns/404.php]: This pattern lost localization.

File: iotix/patterns/header-home.php [hunk 3]
```diff
@@ -1,52 +1,57 @@
...
+<div class="wp-block-group has-link-color" style="padding-top:0;padding-right:0;padding-bottom:0;padding-left:0"><!-- wp:navigation {"ref":5,"overlayBackgroundColor":"primary","overlayTextColor":"tertiary"} /-->
 
 <!-- wp:buttons -->
-<div class="wp-block-buttons"><!-- wp:button {"backgroundColor":"tertiary","textColor":"foreground","fontSize":"small"} -->
-<div class="wp-block-button has-custom-font-size has-small-font-size"><a class="wp-block-button__link has-foreground-color has-tertiary-background-color has-text-color has-background wp-element-button"><?php echo esc_html__( 'Sign In', 'iotix' ); ?></a></div>
+<div class="wp-block-buttons"><!-- wp:button {"backgroundColor":"background","textColor":"primary","className":"is-style-fill","fontSize":"small"} -->
+<div class="wp-block-button has-custom-font-size is-style-fill has-small-font-size"><a class="wp-block-button__link has-primary-color has-background-background-color has-text-color has-background wp-element-button">Sign In</a></div>
```
Comment (pbking) [inline iotix/patterns/header-home.php]: This pattern also lost localization.

File: iotix/patterns/home.php [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+<!-- wp:group {"align":"wide","layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide"><!-- wp:columns {"align":"wide"} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"width":"50%","style":{"border":{"width":"0px","style":"none"}}} -->
+<div class="wp-block-column" style="border-style:none;border-width:0px;flex-basis:50%"><!-- wp:group {"style":{"border":{"radius":"30px"},"spacing":{"padding":{"top":"var:preset|spacing|70","right":"var:preset|spacing|70","bottom":"var:preset|spacing|70","left":"var:preset|spacing|70"}}},"backgroundColor":"tertiary","layout":{"type":"constrained"}} -->
+<div class="wp-block-group has-tertiary-background-color has-background" style="border-radius:30px;padding-top:var(--wp--preset--spacing--70);padding-right:var(--wp--preset--spacing--70);padding-bottom:var(--wp--preset--spacing--70);padding-left:var(--wp--preset--spacing--70)"><!-- wp:heading {"level":3} -->
+<h3 class="wp-block-heading"><strong>No Code Necessary</strong></h3>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>Not a programmer? Use our intuitive, designer-first UI.</p>
```
Comment (pbking) [inline iotix/patterns/home.php]: Should be localized

**Comment by beafialho** [comment]
Oops, lots of lost localizations 😕  Is there a way I can avoid this happening in the future?

File: iotix/patterns/home.php (hunk 4 above)
Comment (jffng) [reply iotix/patterns/home.php]: Done in [4484130](https://github.com/Automattic/themes/pull/7162/commits/448413078d811b785209163bfe6a24329d0db7d9)

File: iotix/parts/footer.html (hunk 1 above)
Comment (jffng) [reply iotix/parts/footer.html]: Done in [a2ddcbe](https://github.com/Automattic/themes/pull/7162/commits/a2ddcbe6724c0bb4d4570c528d027c0efd82ed91)

File: iotix/patterns/header-home.php (hunk 3 above)
Comment (jffng) [reply iotix/patterns/header-home.php]: Done in [3e1548f](https://github.com/Automattic/themes/pull/7162/commits/3e1548fbdc985b0009abb86387847da5a8fea91f)

File: iotix/patterns/404.php (hunk 2 above)
Comment (jffng) [reply iotix/patterns/404.php]: Done in [3e1548f](https://github.com/Automattic/themes/pull/7162/commits/3e1548fbdc985b0009abb86387847da5a8fea91f)

**Comment by jffng** [comment]
> Is there a way I can avoid this happening in the future?

I think we just need to make sure this is happening correctly on theme export from CBT.

**Review by jffng — APPROVED** [review]

**Pull request merged**